src/mentors.json

```diff
@@ -2710,5 +2710,34 @@
         "id": "_gmaiolo"
       }
     ]
+  },
+  {
+    "id": "sainathbanuru96@gmail.com",
+    "name": "sainath",
+    "avatar": "https://i.ibb.co/7KG4Rfq/profile.jpg",
+    "title": "Software Developer",
+    "description": "Mobile App Developer (React Native)",
+    "country": "IN",
+    "tags": [
+      "JavaScript",
+      "React.js",
```

Comment:
Please use the [convention](https://github.com/Coding-Coach/find-a-mentor#synonyms-tags)

```suggestion
      "reactjs",
```